README.md

```diff
@@ -1,5 +1,347 @@
+URL is on the live production site (such as quansight.com), then you will be
+able to safely assume that you're seeing content that was **published** in
+Storyblok at the time the `main` branch was deployed to production. The other
+reason for this discipline is that it's better to limit reviewers and content
+editors to previewing draft content against the `develop` branch in order to
+help catch any potential code/content conflicts before merging to `main`.
+
+When viewing the site in local development or at a Vercel preview URL, there
+should be a banner that explains that you are looking at a preview of the site.
+This helps reduce confusion when screenshots are shared. This banner should not
+be present on a production build of the site.
+
+The preview/development environment banner should allow the user to toggle
+between seeing published versus draft/saved Storyblok content. The banner should
+change colors to indicate which of the two content preview modes the user is in.
+
+The above should hold _except_ when the user is using the Storyblok web
+interface; then the website should always be in "draft" mode, showing saved (but
+not published) content. And as mentioned previously, it should show this content
+against the latest code from the `develop` branch (not the main branch).
+Related: the content team shouldn't have to think about which URL to use with
+the Storyblok editor. It should be just one default URL, and this URL should
+show draft content against the latest code in `develop`.
+
+When a reviewer visits a Vercel preview URL, the site should show draft content
+by default. This, again, is to help catch any potential content/code conflicts
+early. The banner at the top of the site will allow the reviewer to easily
+switch the content mode from published to draft.
+
+Hitting the publish button in Storyblok should not push that content to the
+public-facing site; rather, it should queue up the content for the next
+production deploy. This prevents bypassing any GitHub workflows that have been
+set up for quality control.
+
+Content that is marked as "published" in Storyblok should be ready to be pushed
+to production at any time. Likewise, code that is merged into the `develop`
+branch on GitHub should also be ready to be pushed to `main` at any time.
+Respecting this discipline should allow anyone to deploy a new version of the
+site to production at any time.
+
+All production deploys should happen via commits or merges to the `main` branch.
+Vercel is opinionated about this. We should follow the conventions and opinions
+of the systems we integrate. If there is a content change that needs to bypass
+code changes in the `develop` branch, a pull request can be made to simply
+update a log file. That single commit can then be merged into the `main` branch
+to kick off a deploy. This is a hot fix for content. A hot fix for code can be
+done in a similar way.
```

Comment:
Any such hotfixes should then be back-merged to `develop`, right?

Reply:
yes I think so